### PR TITLE
Move election manager data files into fixtures

### PR DESCRIPTION
**Pull request by carolinemodic**
Moves the data files from election manager into fixtures and restructures the fixtures folder structure a bit. I added a new export object in fixtures that will include references to the file path folders for the CVR and SEMs files. 

Snapshot test changes in election manager are because the version of the either neither election json in fixtures had a slightly different contest ordering. 

@eventualbuddha In order to get the file paths to resolve properly I had to copy all of the csv/cvr files into the /dist folder of fixtures since thats where __dirname from fixtures resolves to, I added a postbuild step in the package.json for fixtures which fixed this for me locally but didn't work automatically with circleci so I had to add buidling fixtures before running the election manager steps on circleci. This feels like the wrong solution to me but I'm not really familiar enough with best practices/paradigms for circleci to know the most appropriate solution. The circle ci changes are in the third commit of this PR. 

Marking this PR as a draft due to the above issue.

### CODE REVIEW

File: apps/election-manager/src/App.test.tsx [hunk 1]
```diff
@@ -33,32 +33,22 @@ import sleep from './utils/sleep'
...
 
 const EITHER_NEITHER_CVR_PATH = join(
-  __dirname,
-  '..',
-  'test',
-  'fixtures',
-  'eitherneither-election',
-  'eitherneither-cvrs.txt'
+  electionWithMsEitherNeitherInternal.cvrDataFolderPath!,
+  'standard.txt'
```
Comment (eventualbuddha) [inline apps/election-manager/src/App.test.tsx:40]: I know these files already existed, but shouldn't they have `.jsonl` extensions?

File: apps/election-manager/test/renderInAppContext.tsx [hunk 2]
```diff
@@ -3,10 +3,7 @@ import React, { RefObject } from 'react'
 import { Router } from 'react-router-dom'
 import { render as testRender } from '@testing-library/react'
 import type { RenderResult } from '@testing-library/react'
-import * as fs from 'fs'
-import { sha256 } from 'js-sha256'
-import { join } from 'path'
-import { parseElection } from '@votingworks/types'
+import { electionWithMsEitherNeitherInternal } from '@votingworks/fixtures'
```
Comment (eventualbuddha) [inline apps/election-manager/test/renderInAppContext.tsx:6]: Why the "internal" name?

File: apps/election-manager/test/renderInAppContext.tsx [hunk 3]
```diff
@@ -24,22 +21,14 @@ import CastVoteRecordFiles, {
...
-)
+const eitherNeitherElection =
+  electionWithMsEitherNeitherInternal.electionDefinition.election
 
-export const defaultElectionDefinition = {
+export const eitherNeitherElectionDefinition = {
   election: eitherNeitherElection,
-  electionData: eitherNeitherElectionData,
-  electionHash: eitherNeitherElectionHash,
+  electionData: JSON.stringify(eitherNeitherElection),
```
Comment (eventualbuddha) [inline apps/election-manager/test/renderInAppContext.tsx:29]: We shouldn't be generating this separately from the hash or they'll be out of sync.

**Comment by carolinemodic** [comment]
@eventualbuddha Back to you, I moved things to .ts files like you suggested. However, I do still want the original files so that they can be used for ui/manual testing and are in one easy to find place so I ended up duplicating the files.

File: libs/fixtures/src/data/electionWithMsEitherNeither/semsFiles/standard.csv.ts [hunk 4]
```diff
@@ -0,0 +1,559 @@
+/* eslint-disable prettier/prettier */
```
Comment (eventualbuddha) [inline libs/fixtures/src/data/electionWithMsEitherNeither/semsFiles/standard.csv.ts:1]: This is probably better done with a glob entry in `.eslintignore`.

File: libs/fixtures/tsconfig.json [hunk 5]
```diff
@@ -68,7 +68,7 @@
     "skipLibCheck": true /* Skip type checking of declaration files. */,
     "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */
   },
-  "include": ["src", "src/data/*.json"],
+  "include": ["src", "src/data/*.json", "src/data/**/*.json"],
```
Comment (eventualbuddha) [inline libs/fixtures/tsconfig.json:71]: `"src/data/**/*.json"` includes `"src/data/*.json"`, so you can just replace the latter with the former.

**Review by eventualbuddha — APPROVED** [review]
LGTM once we have a stronger guarantee the `.ts` and non-`.ts` won't drift apart.

**Comment by carolinemodic** [comment]
Added tests to make sure the original and typescript files stay in sync

**Pull request merged**